ams/optimizer/spark-optimizer/src/main/java/com/netease/arctic/optimizer/spark/SparkExecutor.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netease.arctic.optimizer.spark;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.util.concurrent.ThreadFactoryBuilder;
+import com.netease.arctic.ams.api.OptimizingTask;
+import com.netease.arctic.ams.api.OptimizingTaskResult;
+import com.netease.arctic.optimizer.common.AbstractOptimizerOperator;
+import com.netease.arctic.optimizing.RewriteFilesInput;
+import com.netease.arctic.optimizing.TableOptimizing;
+import com.netease.arctic.utils.SerializationUtil;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.apache.thrift.TException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.concurrent.ArrayBlockingQueue;
+import java.util.concurrent.ThreadFactory;
+import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+public class SparkExecutor extends AbstractOptimizerOperator {
```

Comment:
`SparkExecutor` is a misleading name, easily leading one to think of the executor process in the Spark architecture. 

Based on the code logic, it is suggested to change it to `SparkOptimizingTaskSubmitter`.